test/performance/odp_l2fwd.c

```diff
@@ -6,6 +6,14 @@
  * SPDX-License-Identifier:     BSD-3-Clause
```

Comment:
Unrelated, but should we ditch the "example" l2fwd which is just a symbolic link and just have the "performance" l2fwd?

Reply:
Agreed, I updated the card regarding example application clean up.